JavaScriptResource.mustache

```diff
@@ -1,4 +1,4 @@
-﻿fiftyoneDegreesManager = function() {
+fiftyoneDegreesManager = function() {
```

Comment:
please check if you have accidentally added a BOM character back, as it was removed in one of the previous commits

Reply:
Just checked, and the character's byte sequence `EF BB BF` is not present at the start of the file.
```
> head -c5 JavaScriptResource.mustache | hexdump -C
00000000  66 69 66 74 79                                    |fifty|
00000005
```